src/Main.java

```diff
@@ -24,36 +21,39 @@ public static void main(String[] args) {
                     item.setPrice(scanner.nextDouble());
                     System.out.println("What is the quantity of this item?");
                     item.setQuantity(scanner.nextInt());
+                    id++;
                     item.setId(id);
-                    inventory.addItem();
+                    inventory.addItem(item);
                     break;
                 case 2:
-                    System.out.println("Choose which item(s) you would like to sell.");
-                    inventory.findItem(scanner.nextLine());
-                    System.out.println("What is the quantity you would like to sell?");
+                    inventory.showItems();
+                    inventory.sellItem();
                     break;
 
                 case 3:
+                    inventory.showItems();
```

Comment:
`showItems` is called over and over and over again.  Probably should just be before the switch statement.